modules/data-entry/src/main/frontend/webpack.config.js

```diff
@@ -17,6 +17,9 @@ module.exports = {
     [module_name + 'question']: './src/questionnaire/Question.jsx',
     [module_name + 'answer']: './src/questionnaire/Answer.jsx',
     [module_name + 'booleanQuestion']: './src/questionnaire/BooleanQuestion.jsx',
+    [module_name + 'assignmentIndIcon']: '@material-ui/icons/AssignmentInd.js',
```

Comment:
I would prefer that we use as names what the icon is for, not its official name.

```suggestion
    [module_name + 'subjectsIcon']: '@material-ui/icons/AssignmentInd.js',
```